components/core/utils/test_method_utils.py

```diff
@@ -0,0 +1,53 @@
+from managers.DirectoryManager import Folder, File
+
+
+def username():
```

Comment:
Why are there methods that return a single value?

Reply:
@JaDogg this kind of convention was being used already for functional tests to pass parameters to functions, so I continued with it. Of course, we can remove this and directly pass hard values as well. Thoughts?

Reply:
Why not just constants @vivonk 🤔

Reply:
You can just use these values in the test itself I think.